filter.go

```diff
@@ -39,6 +41,89 @@ func (filter *U32) Type() string {
 	return "u32"
 }
 
+type FilterFwAttrs struct {
+	ClassId   uint32
+	InDev     string
+	Mask      uint32
+	Index     uint32
+	Buffer    uint32
+	Mtu       uint32
+	Mpu       uint16
+	Rate      uint32
+	AvRate    uint32
+	PeakRate  uint32
+	Action    int32
```

Comment:
generally i try to keep the signed ints at this layer as "int" instead of int32 but that is minor